legal/index.md

```diff
@@ -61,11 +61,11 @@ Though we seek to partner with quality companies, we are not responsible for the
 
 ### Offsite Links
 
-As a convenience to Users, the The Monero Project website and services may provide links to other sites or resources. Because we do not review or have no control over such sites and resources, The Monero Project shall not be responsible or liable for use of or reliance on any content, products, services or information at such sites or resources. Inclusion of any links does not imply any endorsement, affiliation, approval, association or sponsorship by The Monero Project of the linked websites, resources, their operators or owners. When you select a link, you may be leaving our website. The information available on Third Parties' websites may have certain restrictions on its use or distribution which differ from these TOU.
+As a convenience to Users, the Monero Project website and services may provide links to other sites or resources. Because we do not review or have no control over such sites and resources, The Monero Project shall not be responsible or liable for use of or reliance on any content, products, services or information at such sites or resources. Inclusion of any links does not imply any endorsement, affiliation, approval, association or sponsorship by The Monero Project of the linked websites, resources, their operators or owners. When you select a link, you may be leaving our website. The information available on Third Parties' websites may have certain restrictions on its use or distribution which differ from these TOU.
```

Comment:
```suggestion
As a convenience to Users, The Monero Project website and services may provide links to other sites or resources. Because we do not review or have no control over such sites and resources, The Monero Project shall not be responsible or liable for use of or reliance on any content, products, services or information at such sites or resources. Inclusion of any links does not imply any endorsement, affiliation, approval, association or sponsorship by The Monero Project of the linked websites, resources, their operators or owners. When you select a link, you may be leaving our website. The information available on Third Parties' websites may have certain restrictions on its use or distribution which differ from these TOU.
```
"the" in "The Monero Project" should be capitalized